tests/test_electrolyzer.py

```diff
@@ -151,6 +155,19 @@ def test_electrolyzer_inequality(self):
                 results["Electrolyzer_fc66.ElectricityIn.Power"] * a[i] + b[i] + 1.0e-3,
             )
 
+        # Check electrolyzer input power
+        np.testing.assert_allclose(
+            results["Electrolyzer_fc66.ElectricityIn.Power"],
+            [1.00000000e08, 1.00000000e08, 1.00000000e08],
+            atol=1e-4,
+        )
+        # Check electrolyzer output massflow
+        np.testing.assert_allclose(
+            results["Electrolyzer_fc66.Gas_mass_flow_out"],
+            [431.367058, 431.367058, 431.367058],
```

Comment:
I think this mass flow is a bound or not? then it is easier to check with the bound even.

Reply:
The bound is 20kg/s

Reply:
is this value a bound on the gas demand? Or anywhere linked to a bound? If yes, take the value from that bound otherwise it is fine.

Reply:
This value is not related to any bound. It is just the output provided by the electrolyzer for 100MW input power